include/jemalloc/internal/tcache_externs.h

```diff
@@ -70,6 +74,8 @@ void tcache_prefork(tsdn_t *tsdn);
 void tcache_postfork_parent(tsdn_t *tsdn);
 void tcache_postfork_child(tsdn_t *tsdn);
 void tcache_flush(tsd_t *tsd);
+void tcache_bin_info_compute(
+    cache_bin_info_t tcache_bin_info[TCACHE_NBINS_MAX]);
```

Comment:
This function is made non-static mainly to use it in the tests in tcache_max.c. Is it worth it? Any other work around?

Reply:
Consider adding a "JET_EXTERN", similar to "JET_MUTABLE":
https://github.com/jemalloc/jemalloc/blob/72cfdce71806443f4ccdbfe10aa5d50346a3d07e/include/jemalloc/internal/jemalloc_internal_macros.h#L38-L42